src/nvim/if_cscope.c

```diff
@@ -933,12 +934,14 @@ static int cs_find(exarg_T *eap)
  *
  * common code for cscope find, shared by cs_find() and do_cstag()
  */
-static int cs_find_common(char *opt, char *pat, int forceit, int verbose, int use_ll, char_u *cmdline)
+static int cs_find_common(
+  char *opt, char *pat, int forceit,
+  int verbose, int use_ll, char_u *cmdline
```

Comment:
Thanks for breaking these, but this isn't the style that we use IIRC, as shown here:

http://neovim.org/develop/style-guide.xml#Function_Declarations_and_Definitions